bin/devil-bot-rust-cdk.ts

```diff
@@ -7,3 +7,7 @@ const app = new cdk.App();
 new DevilBotRustCdkStack(app, 'DevilBotRustCdkStack', {
     env: { region: app.node.tryGetContext('region') || 'us-east-1' },
 });
+// Dev stage
```

Comment:
If this is marked as the dev stage, probably should add a comment above the other stack that says "prod stage". Also, we may want to update the prod stage to a region closer to us like `us-west-1` (northern California).

Reply:
This is doable. I originally only planned for others to just use their prod stage for their own testing purposes. My dev stage is just for the ugly shrek harvey DevilBot Rhett bot and the prod stage was the nice presentable one. I am also considering adding an environment variable as a boolean that tracks if this is a dev or prod version. The prod version interacts with users and I don't want the dev version to do that. Especially everybody's prod and dev versions. We only want my prod version to interact in the different channels and directly with a user